ui/app/templates/components/oidc/key-form.hbs

```diff
@@ -3,24 +3,30 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<PageHeader as |p|>
-  <p.top>
-    <Hds::Breadcrumb>
-      {{#if @model.isNew}}
-        <Hds::Breadcrumb::Item @text="Keys" @route="vault.cluster.access.oidc.keys" />
-      {{else}}
-        <Hds::Breadcrumb::Item @text="Details" @route="vault.cluster.access.oidc.keys.key.details" @model={{@model.name}} />
-      {{/if}}
-      <Hds::Breadcrumb::Item @text="{{if @model.isNew 'Create' 'Edit'}} Key" @current={{true}} />
-    </Hds::Breadcrumb>
-  </p.top>
-  <p.levelLeft>
-    <h1 class="title is-3" data-test-oidc-key-title>
-      {{if @model.isNew "Create" "Edit"}}
-      Key
-    </h1>
-  </p.levelLeft>
-</PageHeader>
+{{#unless @limitOptions}}
```

Comment:
I have conflicting feelings about using a seemingly unrelated arg for hiding/showing the page header. Could we move the page header out of this component into the route template of wherever this component is rendered instead?

Reply:
++ to chelsea's suggestion here. I think to reinforce our newer pattern, this header should move to the [create](https://github.com/hashicorp/vault/blob/main/ui/app/templates/vault/cluster/access/oidc/keys/create.hbs#L10) and [edit](https://github.com/hashicorp/vault/blob/main/ui/app/templates/vault/cluster/access/oidc/keys/key/edit.hbs#L6)